messages/run-command.md

```diff
@@ -1,39 +1,90 @@
 # command.summary
 
-*DRAFT*: Command summary
+Analyze your code with a selection of rules to ensure good coding practices.
 
 # command.description
 
-*DRAFT*: Command description
+Use the --rule-selector flag to narrow down the rules that are used in the analysis based on specific criteria. For example, you can filter by engine, such as the rules associated with the RetireJS or ESLint engine. Or filter by the severity of the rules, such as high or moderate. Every rule has a tag, such as Recommended, and a name, and you can further filter using both.
```

Comment:
I've had a long day at this point... maybe we can continue this review in person. I didn't realize I was so opinionated (as show by all my comments on the rules command). :-/ I think it'll be faster in a meeting.

Reply:
sure, let's chat soon.

Reply:
Invite me to that one too please.